### DISCUSSION TITLE
Schematics

### DISCUSSION THREAD

**Discussion by marmeladapk** [sinara-hw/EEM_FMC_Carrier#5] [Announcements]
Here's alpha version of schematics so that we're on the same page.

[EEM FMC Carrier.PDF](https://github.com/sinara-hw/EEM_FMC_Carrier/files/6810647/EEM.FMC.Carrier.PDF)

**Comment by marmeladapk** [comment]
Recent version of schematics:
[EEM FMC Carrier.PDF](https://github.com/sinara-hw/EEM_FMC_Carrier/files/7123451/EEM.FMC.Carrier.PDF)

**Comment by dhslichter** [comment]
@marmeladapk any updates on designs?

**Reply by marmeladapk** [reply]
There are no significant changes on schematics, only some pin-swapping. PCB is nearly finished.

**Comment by giozara** [comment]
@marmeladapk Hi, what's the latest on the designs?

**Comment by gkasprow** [comment]
The PCB design is currently under review. Next month will go for production

**Reply by giozara** [reply]
Great, thanks!

**Comment by giozara** [comment]
@gkasprow Has the hardware arrived? What's the status on this?

**Reply by giozara** [reply]
@marmeladapk ?

**Reply by gkasprow** [reply]
There was a delay caused by chip shortage. I'll check the status. the company was in holiday model last month.

**Reply by gkasprow** [reply]
we will have prototypes assembled in 10 days.
